core_backend/app/users/models.py

```diff
@@ -191,6 +192,19 @@ async def get_user_by_api_key(
         raise UserNotFoundError("User with given token does not exist.") from err
 
 
+async def get_all_users(
+    asession: AsyncSession,
+) -> Sequence[UserDB]:
```

Comment:
It's because MyPY complains since  `result.scalars().all()` returns a Sequence instead of a list so if we did:
`[user for users in result.scalars().all()] ` it would make sense to use `list`